components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90

```diff
@@ -2210,13 +2233,16 @@ subroutine mam_amicphys_1subarea_clear(        &
 
       mtoo_renamexf(:) = 0
       mtoo_renamexf(nait) = nacc
-
+      if ( strat_accum_coarse_rename ) then
+         mtoo_renamexf(nacc) = ncrs 
```

Comment:
Maybe I'm missing something, but I don't see where `ncrs` is ever initialized in this file.